angular.json

```diff
@@ -150,12 +150,12 @@
             "styles": [
               {
                 "input": "apps/sam-design-system-site/src/uswds-styles.scss",
-                "inject": false,
+                "inject": true,
```

Comment:
Can you try building on federalist with these as false? They should not be required to build successfully.

Reply:
it i make false still failing

Reply:
Just looked, it's failing because the css file that is created when building is named slightly differently than what is expected by the stackblitz generation script (sds.a1b2c3.css vs sds.css). Just need to update that script to expect the name that does not include a chunk number and it'll pass, and we can have the index.html not include 2 references to the same file and 1 to a file not even used.

Take a look at this branch I made off yours to see the diff and it passing: https://github.com/GSA/sam-design-system/compare/test